components/PostCard.js

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import Link from 'next/link';
+// import { deletePost } from '../../utils/data/postData';
 
-export default function PostCard() {
+export default function PostCard({ postObj }) {
```

Comment:
Can you change the postObj to accept individual props, like so: { id, title, content, publicationDate }? that way we'll be able to avoid using snake case in the component. If you do this, you'll need to change the proptypes and the way the props are handled in the component (see complete changes below):

  `import React from 'react';
  import PropTypes from 'prop-types';
  import { Card } from 'react-bootstrap';
  import Button from 'react-bootstrap/Button';
  import Link from 'next/link';
  // import { deletePost } from '../../utils/data/postData';
  
  // eslint-disable-next-line no-unused-vars
  export default function PostCard({
    id, title, content, publicationDate, onUpdate,
  }) {
    // const deleteThisPost = () => {
    //   if (window.confirm(`Delete ${title}?`)) {
    //     deletePost(id).then(() => onUpdate());
    //     window.location.reload();
    //   }
    // };
    return (
      <>
        <Card className="text-center">
          <Card.Body>
            <Card.Title>{title}</Card.Title>
            <Card.Text>{content}</Card.Text>
            <Card.Text>{publicationDate}</Card.Text>
            <Link href={`/posts/edit/${id}`} passHref>
              <Button variant="info">EDIT</Button>
            </Link>
            {/* <Button variant="danger" onClick={deleteThisPost} className="m-2">
              DELETE
            </Button> */}
          </Card.Body>
          <Card.Footer />
        </Card>
      </>
    );
  }
  
  PostCard.propTypes = {
    id: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    publicationDate: PropTypes.number.isRequired,
    content: PropTypes.string.isRequired,
    approved: PropTypes.bool.isRequired,
    onUpdate: PropTypes.func.isRequired,
  };
`